frontend/src/components/Overview/CommunityNews.vue

```diff
@@ -28,17 +52,53 @@
         </b-card-body>
       </b-card>
     </div>
+    --></b-card>
   </div>
 </template>
 <script>
 import News from '@/assets/News/news.json'
+
+// Berechne den heutigen Tag des Jahres
+const jetzt = new Date()
+const start = new Date(jetzt.getFullYear(), 0, 0)
+const diff = jetzt - start + (start.getTimezoneOffset() - jetzt.getTimezoneOffset()) * 60 * 1000
+const tag = Math.floor(diff / (1000 * 60 * 60 * 24))
+
+// Berechne Faktor1 und Faktor2
+const faktor1 = Math.pow(0.998098, tag)
+const faktor2 = 1 / faktor1
+
 export default {
   name: 'CommunityNews',
   data() {
     return {
       News,
+      tag,
+      faktor: faktor2.toFixed(2),
+      gradido: '',
+      dankbar: '',
+      ergebnis: '',
     }
   },
+  methods: {
+    umrechnen() {
+      if (this.gradido) {
+        const ergebnis = this.gradido * faktor2
+        this.dankbar = ergebnis.toFixed(2)
+      } else if (this.dankbar) {
+        const ergebnis = this.dankbar * faktor1
+        this.gradido = ergebnis.toFixed(2)
+      } else {
+        this.ergebnis = 'Bitte geben Sie entweder Gradido oder DankBar ein'
+      }
```

Comment:
Can be written with a cleaner way make the formular interactiv if no value filed then error else give result